CONTRIBUTING.md

```diff
@@ -23,6 +23,105 @@ First of all, thank you for contributing. It’s appreciated.
 
 ## Component structure
 
+### Create a new Component
+Here are some basic steps to get you started building your first component.
+
+1. Create your new component folder
+`src/components/MyNewComponent/`
+2. Create your component file(s)
+```
+    /MyNewComponent/
```

Comment:
Some other optional files that consumers may wish to create:
- MyNewComponent.reducers.js
- MyNewComponent.reducers.spec.js
- MyNewComponent.constants.js
- MyNewComponent.selectors.js
- MyNewComponent.selectors.spec.js